package.json

```diff
@@ -4,65 +4,63 @@
   "version": "0.0.1",
   "type": "module",
   "scripts": {
-    "dev:android": "ionic cap run android -l --external",
-    "dev:android:device": "ionic cap run android -l --external --target=$ANDROID_DEVICE",
-    "dev:ios": "ionic cap run ios -l --external",
-    "dev:ios:device": "ionic cap run ios -l --external --target=$IOS_DEVICE",
+    "dev:android": "ionic cap run android -l --external --no-reload",
```

Comment:
does that mean we can't have hot reload anymore? Seemed quite useful for development no?

Reply:
Ok I see ok that's a pain yes, thanks for changing that